docs/development/visualize/development-create-visualization.asciidoc

```diff
@@ -404,10 +438,30 @@ You can access filter bar and time picker through the objects defined on `vis.AP
 ==== timeFilter
 
 Update the timefilter time values and call update() method on it to update time picker
+
 ["source","js"]
 -----------
    timefilter.time.from = moment(ranges.xaxis.from);
    timefilter.time.to = moment(ranges.xaxis.to);
    timefilter.time.mode = 'absolute';
    timefilter.update();
------------
\ No newline at end of file
+-----------
+
+
+[[development-aggconfig]]
+=== AggConfig object
+ 
+Few times we mentioned the AggConfig object and you will probably have to handle with it in every visualization you 
+will create. Here we provide short description of some of the methods on it, however the best reference would be 
+to actually check the code and other visualizations.
```

Comment:
I'd say something like:

`The AggConfig object represents an aggregation search to Elasticsearch, plus some additional functionality to manage data-values that belong to this aggregation. This is primarily used internally in Kibana, but you may find you have a need for it when writing your own visualization. Here we provide short description of some of the methods on it, however the best reference would be to actually check the source code.`